scripts/compose.py

```diff
@@ -418,9 +418,18 @@ def __init__(self, **options):
             "enable_elasticsearch", True) else {}
         self.build = self.options.get("apm_server_build")
 
-        if self.at_least_version("7.2") and not self.oss and not self.options.get("apm_server_ilm_disable"):
+        if self.options.get("apm_server_ilm_disable"):
+            self.apm_server_command_args.append(("apm-server.ilm.enabled", "false"))
+        elif self.at_least_version("7.2") and not self.at_least_version("7.3") and not self.oss:
             self.apm_server_command_args.append(("apm-server.ilm.enabled", "true"))
 
+        if self.options.get("disable_acm"):
+            self.apm_server_command_args.append(("apm-server.kibana.enabled", "false"))
+        elif self.at_least_version("7.3"):
+            self.apm_server_command_args.extend([
+                ("apm-server.kibana.enabled", "true"),
+                ("apm-server.kibana.host", "kibana:5601")])
```

Comment:
let's extract `kibana:5601` and use it for `setup.kibana.host` as well - we probably want to add yet another command line option for this, but later